test-packages/test-services-odata-v2/generate-odata-services.ts

```diff
@@ -1,19 +1,19 @@
 import { resolve, join } from 'path';
 import { createLogger } from '@sap-cloud-sdk/util';
-import { generate } from '@sap-cloud-sdk/generator';
+import { generate, GeneratorOptions } from '@sap-cloud-sdk/generator';
 
 const logger = createLogger('generate-odata-services');
 
-const generatorConfigOData = {
+const generatorConfigOData: Partial<GeneratorOptions> = {
   overwrite: true,
   transpile: true,
   useSwagger: false,
   readme: false,
   clearOutputDir: false,
   packageJson: false,
-  generateSdkMetadata: false,
   skipValidation: true,
-  verbose: false
+  verbose: false,
+  generateESM: true
```

Comment:
```suggestion
  generateESM: false
```

Don't set it true here as the generate tests will fail. I believe the default is false anyway, so you can also leave it out.